Cargo.toml

```diff
@@ -18,7 +18,7 @@ pallas = { git = "https://github.com/txpipe/pallas.git", features = ["unstable"]
 gasket = { version = "^0.5", features = ["derive"] }
 # gasket = { path = "../../construkts/gasket-rs/gasket", features = ["derive"] }
 
-utxorpc = { version = "1.0.0-alpha.1" }
+utxorpc = { git = "https://github.com/Mercurial/rust-sdk.git",branch = "feat/update-dependency-spec-0.3.0" }
```

Comment:
until the SDK has anything interesting, I would just depend on `utxorpc-spec = "0.3.0"` and avoid the extra step.

In the near future, when the SDK has some actual benefits other than a re-export of structures, we'll switch back.

Reply:
that makes sense, will do the needed change!